src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -65,6 +68,9 @@ public Command parseCommand(String userInput) throws ParseException {
         // Lower level log messages are used sparingly to minimize noise in the code.
         logger.fine("Command word: " + commandWord + "; Arguments: " + arguments);
 
+        isClearCommand = confirmClearCommand;
+        confirmClearCommand = false;
```

Comment:
should this start with is... since it is a boolean?

Reply:
Okay, fixed!